src/runtime/containerd-shim-v2/service.go

```diff
@@ -58,8 +58,12 @@ var (
 // concrete virtcontainer implementation
 var vci vc.VC = &vc.VCImpl{}
 
+// shimLog is logger for shim package
+var shimLog = logrus.WithField("source", "containerd-kata-shim-v2")
+
 // New returns a new shim service that can be used via GRPC
 func New(ctx context.Context, id string, publisher events.Publisher) (cdshim.Shim, error) {
+	shimLog = shimLog.WithField("ID", id)
```

Comment:
Why do we need two logger here?  why not use  the same one log shimLog here?

Reply:
- If we are creating a new logger, it needs to have all the "standard" fields we expect:

  - https://github.com/kata-containers/tests/tree/master/cmd/log-parser#logfile-requirements

- Admittedly we haven't actually documented this, but we have an informal standard of always using lower-case field names with dashes between fields if necessary, so this would normally be `id` and `eventType` (below) would be `event-type`.